content/en/docs/tasks/administer-cluster/storage-object-in-use-protection.md

```diff
@@ -3,23 +3,23 @@ approvers:
 - msau42
 - jsafrane
 title: Storage Object in Use Protection
-content_template: templates/task
 ---
 
-{{% capture overview %}}
-{{< feature-state for_k8s_version="v1.10" state="beta" >}}
+{% capture overview %}
 
-Persistent volume claims (PVCs) that are in active use by a pod and persistent volumes (PVs) that are bound to PVCs can be protected from pre-mature removal.
+Persistent volume claims (PVCs) that are in active use by a pod and persistent volumes (PVs) that are bound to PVCs can be protected from premature removal.
 
-{{% /capture %}}
+{% endcapture %}
```

Comment:
/assign @chenopis 

Which capture syntax is the right one now? I suspect Hugo fixed this and then changes in this PR are effectively undoing the fix.

Reply:
Post-Hugo, it's the `{{ }}` tags, so in this case it's `{{% /capture %}}`. Yeah, it looks like this PR is reverting the changes needed for Hugo.

Reply:
Cool, so these lines need to be reverted to use the Hugo syntax.